### PR TITLE
Manatime-3: Clients content type.

### PR DESCRIPTION
**Pull request by josue2591**
**Step to tests**
- Run the script manatime_local_install.sh 
- See if the features manatime_clients was added and activated.
- Add a new client node

**DoD**
- As an administrator I should be able to go to node/add/client and add a new client with the described fields
- As an administrator, I should be able to view a client (no themed) and click on the Edit tab and update that client.

### CODE REVIEW

**Comment by kporras07** [comment]
So far it's awesome to see you're getting stuff done :D
I haven't tested this, but before doing that, a couple of comments:
1) We forgot to tell you :( We use to create a field bases feature (manatime_field_bases) in order to group all field bases and avoid any possible dependency hell. Please do it and sorry for letting you know this until now.
2) It's awesome you prefixed the feature module with manatime :D An internal standard that we (again) forgot to tell you is to prefix custom modules with project name (exactly as you did it) and features with project-name_feature_name (manatime_feature_clients). Again, please do it and sorry for letting you know this until now.
3) Wercker build failed due to coding standards, fix it and it will work as expected
4) Please edit the PR title

File: modules/features/manatime_clients/manatime_clients.features.field_instance.inc [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?php
```
Comment (cienvaras) [inline modules/features/manatime_clients/manatime_clients.features.field_instance.inc]: Right now both features are duplicated. The original version (without `feature` in the name) should be deleted.

File: modules/features/manatime_feature_clients/manatime_feature_clients.features.field_instance.inc [hunk 2]
```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @file
+ * manatime_clients.features.field_instance.inc
```
Comment (cienvaras) [inline modules/features/manatime_feature_clients/manatime_feature_clients.features.field_instance.inc]: Fix this comment with the correct filename.

File: modules/features/manatime_feature_clients/manatime_feature_clients.info [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name = manatime_feature_clients
```
Comment (cienvaras) [inline modules/features/manatime_feature_clients/manatime_feature_clients.info]: The _human_ name should be something more easy to understand, like "Manatime clients feature".

File: modules/features/manatime_feature_field_bases/manatime_feature_field_bases.info [hunk 4]
```diff
@@ -0,0 +1,11 @@
+name = manatime_feature_field_bases
```
Comment (cienvaras) [inline modules/features/manatime_feature_field_bases/manatime_feature_field_bases.info]: The _human_ name should be something like "Manatime field bases".

File: modules/features/manatime_field_bases/manatime_field_bases.features.field_base.inc [hunk 5]
```diff
@@ -0,0 +1,60 @@
+<?php
```
Comment (cienvaras) [inline modules/features/manatime_field_bases/manatime_field_bases.features.field_base.inc]: Remove this duplicated feature.

File: settings/settings.php [hunk 6]
```diff
@@ -53,6 +53,10 @@
   'token',
```
Comment (cienvaras) [inline settings/settings.php:53]: Make sure permissions for this file are correct.

File: modules/features/manatime_feature_field_bases/manatime_feature_field_bases.info [hunk 7]
```diff
@@ -0,0 +1,11 @@
+name = Manatime field bases
+core = 7.x
+package = manatime
+version = 7.x-7.0
```
Comment (kporras07) [inline modules/features/manatime_feature_field_bases/manatime_feature_field_bases.info]: 7.x-1.0

File: modules/features/manatime_feature_clients/manatime_feature_clients.info [hunk 8]
```diff
@@ -0,0 +1,23 @@
+name = Manatime clients feature
+description = Content type clients.
+core = 7.x
+package = manatime
```
Comment (kporras07) [inline modules/features/manatime_feature_clients/manatime_feature_clients.info:4]: Add version and project path please.

**Comment by kporras07** [comment]
Overall code looks great! Just 2 minor fixes. I'm proceeding with functional review right now.

File: modules/features/manatime_feature_clients/manatime_feature_clients.info [hunk 9]
```diff
@@ -6,13 +6,11 @@ version = 7.x-1.0
 dependencies[] = ctools
 dependencies[] = features
 dependencies[] = list
+dependencies[] = manatime_feature_field_bases
 dependencies[] = options
 dependencies[] = strongarm
-dependencies[] = text
```
Comment (kporras07) [inline modules/features/manatime_feature_clients/manatime_feature_clients.info]: Re-add this line.

File: modules/features/manatime_feature_clients/manatime_feature_clients.info [hunk 10]
```diff
@@ -6,13 +6,11 @@ version = 7.x-1.0
...
 features[ctools][] = strongarm:strongarm:1
 features[features_api][] = api:2
-features[field_instance][] = node-clients-body
```
Comment (kporras07) [inline modules/features/manatime_feature_clients/manatime_feature_clients.info]: Re-add this line.

File: modules/features/manatime_feature_clients/manatime_feature_clients.info [hunk 11]
```diff
@@ -6,13 +6,11 @@ version = 7.x-1.0
...
-features[field_instance][] = node-clients-field_active
```
Comment (kporras07) [inline modules/features/manatime_feature_clients/manatime_feature_clients.info]: Re-add this line.

**Comment by kporras07** [comment]
@josue2591 assuming Wercker passes, :shipit:

**Pull request merged**